adal/src/main/java/com/microsoft/aad/adal/AuthenticationActivity.java

```diff
@@ -564,6 +571,11 @@ public void onBackPressed() {
     private void cancelRequest() {
         Logger.v(TAG, "Sending intent to cancel authentication activity");
         Intent resultIntent = new Intent();
+
+        if (mUIEvent != null) {
```

Comment:
happened in the cancel test case.

---
In reply to: [90353733](https://github.com/AzureAD/azure-activedirectory-library-for-android/pull/736#discussion_r90353733) [](ancestors = 90353733)